ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -809,7 +823,11 @@ static PrjFS_Result HandleRecursivelyEnumerateDirectoryRequest(const MessageHead
 static PrjFS_Result HandleHydrateFileRequest(const MessageHeader* request, const char* relativePath)
 {
 #ifdef DEBUG
-    cout << "PrjFSLib.HandleHydrateFileRequest: " << relativePath << endl;
+    cout
+        << "PrjFSLib.HandleHydrateFileRequest: "
+        << relativePath
+        << " Process name: " << request->procname
```

Comment:
Also print the `pid` (here and everyone else we're printing `request->procname`)?